cmd/kube-scheduler/app/options/options_test.go

```diff
@@ -146,6 +146,25 @@ users:
 		t.Fatal(err)
 	}
 
+	// plugin config
+	pluginconfigFile := filepath.Join(tmpDir, "plugin.yaml")
+	if err := ioutil.WriteFile(pluginconfigFile, []byte(fmt.Sprintf(`
+apiVersion: kubescheduler.config.k8s.io/v1alpha1
+kind: KubeSchedulerConfiguration
+clientConnection:
+  kubeconfig: "%s"
+leaderElection:
+  leaderElect: true
+plugins:
```

Comment:
How much work would it be to add `pluginConfig` in this test?

Reply:
Thanks, I have added the unit test for the plugin configuration.